src/routes/collectibles/routes.rs

```diff
@@ -165,3 +170,45 @@ pub async fn get_collectibles(
     )
     .await
 }
+
+/// `/v2/chains/<chain_id>/safes/<safe_address>/collectibles?<cursor>&<trusted>&<exclude_spam>` <br />
+/// Returns collectibles paginated
+///
+/// # Collectibles
+///
+/// The collectibles endpoint does not implement any logic in the client-gateway. The response from the core services is cached and then forwarded to the clients.
+///
+/// ## Path
+///
+/// - `/v1/chains/<chain_id>/safes/<safe_address>/collectibles?<cursor>&<trusted>&<exclude_spam>` : Returns a list of the ERC721 tokens stored in a safe
+///
+/// ## Query parameters
+///
+/// `<cursor>` : Pagination parameters (limit and offset)
+/// `<trusted>` : A token is defined as trusted by our core handlers process when adding them. Default value is `false`
+/// `<exclude_spam>`: A token is defined as spam by our core handlers process when adding them. Default value is `true`
+#[openapi(tag = "Collectibles")]
+#[get("/v2/chains/<chain_id>/safes/<safe_address>/collectibles?<cursor>&<trusted>&<exclude_spam>")]
+pub async fn get_collectibles_paginated(
+    context: RequestContext,
+    chain_id: String,
+    safe_address: String,
+    cursor: Option<String>,
+    trusted: Option<bool>,
+    exclude_spam: Option<bool>,
+) -> ApiResult<content::RawJson<String>> {
+    CacheResponse::new(&context, ChainCache::Other)
```

Comment:
I will review this, ChainCache?

Reply:
@fmrsabino what do you think? would be necessary caching the response?

Reply:
I think we should always cache in the cgw, as in the other endpoints

Reply:
> I think we should always cache in the cgw, as in the other endpoints

That's correct. However for the collectibles, the first version of the endpoint only cached the request to the Safe Transaction Service and not the response given by the Safe Client Gateway to the clients.

> @fmrsabino what do you think? would be necessary caching the response?

I would say that we can start by not caching the response to the clients. We might be introducing unnecessary complexity. I'd roll out this endpoint and keep track of its usage.

Reply:
Ok, I agree with only chaching the request.